src/pybind/mgr/dashboard/module.py

```diff
@@ -1102,7 +1139,8 @@ def _osd_summary(self, osd_id, osd_info):
                 result['up'] = osd_info['up']
                 result['in'] = osd_info['in']
 
-                result['url'] = get_prefixed_url("/osd/perf/{0}".format(osd_id))
+                #result['url'] = get_prefixed_url("/osd/perf/{0}".format(osd_id))
+                result['url'] = get_prefixed_url("/perf_counters/osd/{0}".format(osd_id))
```

Comment:
This should still link to /perf/, and then the page at /perf/ should have a link to the /perf_counters/ equivalent.